cli/cmd/cluster-apply.go

```diff
@@ -125,7 +126,12 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 }
 
 func verifyCluster(kubeconfigPath string, expectedNodes int) error {
-	cs, err := k8sutil.NewClientsetFromFile(kubeconfigPath)
+	kubeconfig, err := ioutil.ReadFile(kubeconfigPath) // #nosec G304
```

Comment:
perhaps reading of the kubeconfig file as a function in  `internal` package rather than having the same code across multiple places ?

Reply:
The duplication is intentional, so "target" function (`k8sutil.NewClientset`) is already being called. Next PR will be changing the parameter types we pass around, so `ioutil.ReadFile` call will be removed. Adding function to de-duplicate has little sense here.

See #631 for more details.